DNN Platform/Library/Common/Utilities/StringExtensions.cs

```diff
@@ -65,6 +65,21 @@ public static string ReplaceIgnoreCase(this string source, string oldValue, stri
             }
 
             return Regex.Replace(source, Regex.Escape(oldValue), newValue, RegexOptions.IgnoreCase);
+        }
+
+        /// <summary>
+        /// Removes new line separators.
+        /// </summary>
+        /// <param name="source">The source.</param>
+        /// <returns>The string without new line separators.</returns>
+        public static string RemoveNewLineSeparators(this string source)
+        {
+            if (string.IsNullOrWhiteSpace(source))
+            {
+                return source;

```

Comment:
I would think we'd want a method named `RemoveNewLineSeparators` to still act on a string that is only white-space.  Should this check be `IsNullOrEmpty`?